src/Facebook/InstantArticles/Transformer/Transformer.php

```diff
@@ -444,4 +483,78 @@ public function getDefaultDateTimeZone()
     {
         return $this->defaultDateTimeZone;
     }
+
+    /**
+     * Sets the default style to be applyied to the articles generated from this transformation.
+     *
+     * @param string $defaultStyleName
+     */
+    public function setDefaultStyleName($defaultStyleName)
+    {
+        Type::enforce($defaultStyleName, Type::STRING);
+        $this->defaultStyleName = $defaultStyleName;
+    }
+
+    /**
+     * Gets the default style name for Instant Article generated during transformation.
+     *
+     * @return string
+     */
+    public function getDefaultStyleName()
+    {
+        return $this->defaultStyleName;
+    }
+
+    /**
+     * Applies the settings loaded from the rules.json informed on loadRules method.
+     *
+     * @param InstantArticle The InstantArticle to have the settings set.
+     * @return InstantArticle The article with settings added if needed.
+     */
+    public function handleTransformationSettings($instantArticle)
+    {
+        if (!Type::isTextEmpty($this->getDefaultStyleName())) {
+            $instantArticle->withStyle($this->getDefaultStyleName());
+        }
+
+        if ($this->adsSettings) {
+            $ad = Ad::create();
+            if (!Type::isTextEmpty($this->adsSettings->getAudienceNetworkPlacementId())) {
+                $ad->withSource(
+                    'https://www.facebook.com/adnw_request?placement='.
+                    $this->adsSettings->getAudienceNetworkPlacementId()
+                );
+            }
+            if (!Type::isTextEmpty($this->adsSettings->getRawHTML())) {
+                $ad->withHTML(
+                    $this->adsSettings->getRawHTML()
+                );
+            }
+            $instantArticle->getHeader()->addAd($ad);
+        }
+
+        if ($this->analyticsSettings) {
+            $analytics = Analytics::create();
```

Comment:
Let's also move this logic to AnalyticsSettings (by creating a `getAnalytics` method).